chain/client/tests/process_blocks.rs

```diff
@@ -998,10 +1008,30 @@ fn test_bad_orphan() {
             };
             let chunk = match &mut body.chunks[0] {
                 ShardChunkHeader::V1(_) => unreachable!(),
+                #[cfg(not(feature = "protocol_feature_block_header_v3"))]
                 ShardChunkHeader::V2(chunk) => chunk,
+                #[cfg(feature = "protocol_feature_block_header_v3")]
+                ShardChunkHeader::V2(_) => unreachable!(),
```

Comment:
why is this unreachable?

Reply:
Because if the feature is enabled then the code will only ever create `ShardChunkHeader::V3` in tests.